nodejs/packages/layer/package.json

```diff
@@ -23,32 +23,32 @@
   "author": "OpenTelemetry Authors",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=12.0.0"
   },
   "dependencies": {
-    "@opentelemetry/api": "^1.1.0",
+    "@opentelemetry/api": "^1.2.0",
```

Comment:
The latest tag is still pointed to 1.1.0. we should double check to use the `next` tag other than the `latest` tag.
<img width="784" alt="image" src="https://user-images.githubusercontent.com/41936996/187746367-34f02e88-31a4-4c7e-b983-87ebe06d8da1.png">

Reply:
The [opentelemetry-js-api](https://github.com/open-telemetry/opentelemetry-js-api/releases) shows `1.2.0` to be the `latest` and doesn't call out anywhere to be RC. Is there a reason to still be using `1.1.0` ?

Reply:
Yeah, not sure why it is not targeted to latest in `npm` though. 
<img width="468" alt="image" src="https://user-images.githubusercontent.com/41936996/187748822-dde49fd9-c66d-48e0-a033-365b0508f111.png">

Reply:
Looks like it is safe to use the version `1.2.0`. it is compatible and can confirm from here : https://github.com/open-telemetry/opentelemetry-js/pull/3177